include/ck_tile/ops/fmha/block/page_block_navigator.hpp

```diff
@@ -0,0 +1,244 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/core/tensor/tile_window.hpp"
+
+namespace ck_tile {
+
+// assume that we have only 1 page-block
+struct TrivialPageBlockNavigator
+{
+    using WindowOrigin = multi_index<2>;
+
+    template <typename TensorView, typename WindowLengths>
+    CK_TILE_HOST_DEVICE static constexpr auto make_tile_window(const TensorView& tensor_view,
+                                                               const WindowLengths& window_lengths,
+                                                               const WindowOrigin& window_origin)
+    {
+        return make_tuple(/*block_index=*/0,
+                          ck_tile::make_tile_window(tensor_view, window_lengths, window_origin));
+    }
+
+    template <typename TensorView, typename WindowLengths>
+    CK_TILE_HOST_DEVICE static constexpr auto
+    make_tile_window(const tile_window_with_static_lengths<TensorView, WindowLengths>& tile_window,
+                     const WindowOrigin& window_origin)
+    {
+        return make_tuple(
+            /*block_index=*/0, ck_tile::make_tile_window(tile_window, window_origin));
+    }
+
+    template <typename TensorView, typename WindowLengths, typename TileDistribution>
+    CK_TILE_HOST_DEVICE static constexpr auto
+    make_tile_window(const tile_window_with_static_lengths<TensorView, WindowLengths>& tile_window,
+                     const WindowOrigin& window_origin,
+                     const TileDistribution& tile_distribution)
+    {
+        return make_tuple(
+            /*block_index=*/0,
+            ck_tile::make_tile_window(tile_window, window_origin, tile_distribution));
+    }
+
+    template <typename TileWindow>
+    CK_TILE_HOST_DEVICE static index_t
+    move_tile_window(index_t /*block_index*/,
```

Comment:
Hi, Are there some docs I could know exactly about concetps in CK_tile, such as TileWindow, Navigator, Coordinate transform, Pipeline, etc? Thanks!